com.alibaba:dubbo-demo-api:jar:2.5.9 这个jar包没有

在dubbo-spring-boot-samples中com.alibaba:dubbo-demo-api:jar:2.5.9 这个jar包没有

[ERROR] Failed to execute goal on project dubbo-spring-boot-actuator: Could not resolve dependencies for project com.alibaba.boot:dubbo-spring-boot-actuator:jar:1.0.0-SNAPSHOT: Failure to find com.alibaba:dubbo-demo-api:jar:2.5.9 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

--- a/dubbo-spring-boot-actuator/pom.xml
+++ b/dubbo-spring-boot-actuator/pom.xml
@@ -65,13 +65,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
-            <groupId>com.alibaba</groupId>
-            <artifactId>dubbo-demo-api</artifactId>
-            <version>${dubbo.version}</version>
-            <scope>test</scope>
-        </dependency>
-
     </dependencies>
 
 </project>

--- a/dubbo-spring-boot-actuator/src/test/java/com/alibaba/boot/dubbo/actuate/endpoint/mvc/DubboMvcEndpointTest.java
+++ b/dubbo-spring-boot-actuator/src/test/java/com/alibaba/boot/dubbo/actuate/endpoint/mvc/DubboMvcEndpointTest.java
@@ -19,7 +19,6 @@ package com.alibaba.boot.dubbo.actuate.endpoint.mvc;
 import com.alibaba.boot.dubbo.actuate.endpoint.DubboEndpoint;
 import com.alibaba.boot.dubbo.autoconfigure.DubboAutoConfiguration;
 import com.alibaba.dubbo.config.annotation.Service;
-import com.alibaba.dubbo.demo.DemoService;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -137,7 +136,7 @@ public class DubboMvcEndpointTest {
 
         Assert.assertEquals(1, services.size());
 
-        Map<String, Object> demoServiceMeta = services.get("ServiceBean@com.alibaba.dubbo.demo.DemoService#dubboMvcEndpointTest.DefaultDemoService");
+        Map<String, Object> demoServiceMeta = services.get("ServiceBean@com.alibaba.boot.dubbo.actuate.endpoint.mvc.DubboMvcEndpointTest$DemoService#dubboMvcEndpointTest.DefaultDemoService");
 
         Assert.assertEquals("1.0.0", demoServiceMeta.get("version"));
 
@@ -184,6 +183,10 @@ public class DubboMvcEndpointTest {
             return "Hello, " + name + " (from Spring Boot)";
         }
 
+    }
+
+    interface DemoService {
+        String sayHello(String name);
     }
 
 }

